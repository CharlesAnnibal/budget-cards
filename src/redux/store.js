import { createStore, applyMiddleware } from "redux";
import logger from 'redux-logger';
import rootReducer from "./reducers/root.reducer";
import BudgetReducer from  "./reducers/Budget.reducer"

const middlewares = [logger];

function reducerTeste(){
    return [
            {
                nome:"teste1",
                "amount":"100"
            },
            {
                nome:"teste2",
                "amount":"100"
            },
            {
                nome:"teste3",
                "amount":"100"
            },
     
    ]
}
const store = createStore(rootReducer);
export default store;