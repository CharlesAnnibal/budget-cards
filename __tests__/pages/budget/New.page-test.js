import 'react-native';
import React from 'react';
import NewBudget from '../../../src/pages/budget/New.page';
import store from "../../../src/redux/store";
import { Provider } from "react-redux";

import renderer from 'react-test-renderer';


it('renders correctly', () => {
//const wrapper = () => <Provider><NewBudget/></Provider>
  const newBudgetComponent = renderer.create(<Provider store={store}><NewBudget/></Provider>).toJSON();
  expect(newBudgetComponent).toMatchSnapshot();
});

it('renders new Budget screen correctly, with an empty form', () => {
    //const wrapper = () => <Provider><NewBudget/></Provider>
    const newBudgetComponent = renderer.create(<Provider store={store}><NewBudget amount={55} /></Provider>).toJSON();
    expect(newBudgetComponent).toMatchSnapshot();
});
