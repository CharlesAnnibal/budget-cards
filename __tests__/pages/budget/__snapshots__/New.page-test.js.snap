// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={Object {}}
      >
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Amount"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Description"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Color"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <Text
            placeholder="Date start"
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "width": "85%",
                },
              ]
            }
            uppercase={false}
          >
            Fri Apr 10 2020
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#114acf",
                "borderBottomWidth": null,
                "borderColor": "#c2ffcd",
                "borderLeftWidth": null,
                "borderRadius": 10,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": 2,
                "elevation": 2,
                "flexDirection": "row",
                "height": "80%",
                "justifyContent": "center",
                "marginVertical": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": undefined,
                "shadowOffset": undefined,
                "shadowOpacity": undefined,
                "shadowRadius": undefined,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 24,
                      "marginLeft": 16,
                      "marginRight": 16,
                      "paddingTop": 2,
                    },
                  ],
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              type="FontAwesome"
            >
              ÔÅ≥
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <Text
            placeholder="Date start"
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "width": "85%",
                },
              ]
            }
            uppercase={false}
          >
            Fri Apr 10 2020
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#114acf",
                "borderBottomWidth": null,
                "borderColor": "#c2ffcd",
                "borderLeftWidth": null,
                "borderRadius": 10,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": 2,
                "elevation": 2,
                "flexDirection": "row",
                "height": "80%",
                "justifyContent": "center",
                "marginVertical": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": undefined,
                "shadowOffset": undefined,
                "shadowOpacity": undefined,
                "shadowRadius": undefined,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 24,
                      "marginLeft": 16,
                      "marginRight": 16,
                      "paddingTop": 2,
                    },
                  ],
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              type="FontAwesome"
            >
              ÔÅ≥
            </Text>
          </View>
        </View>
      </View>
      <View
        accessible={true}
        focusable={true}
        forwardedRef={[Function]}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#114acf",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 2,
            "flexDirection": "row",
            "height": 45,
            "justifyContent": "center",
            "opacity": 1,
            "paddingBottom": 6,
            "paddingTop": 6,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            }
          }
          uppercase={false}
        >
          Save Budget
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;

exports[`renders new Budget screen correctly, with an empty form 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={Object {}}
      >
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Amount"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Description"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            placeholder="Color"
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 5,
                "paddingRight": 5,
                "top": 1.5,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
          />
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <Text
            placeholder="Date start"
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "width": "85%",
                },
              ]
            }
            uppercase={false}
          >
            Fri Apr 10 2020
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#114acf",
                "borderBottomWidth": null,
                "borderColor": "#c2ffcd",
                "borderLeftWidth": null,
                "borderRadius": 10,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": 2,
                "elevation": 2,
                "flexDirection": "row",
                "height": "80%",
                "justifyContent": "center",
                "marginVertical": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": undefined,
                "shadowOffset": undefined,
                "shadowOpacity": undefined,
                "shadowRadius": undefined,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 24,
                      "marginLeft": 16,
                      "marginRight": 16,
                      "paddingTop": 2,
                    },
                  ],
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              type="FontAwesome"
            >
              ÔÅ≥
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "opacity": 1,
            }
          }
        >
          <Text
            placeholder="Date start"
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "width": "85%",
                },
              ]
            }
            uppercase={false}
          >
            Fri Apr 10 2020
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#114acf",
                "borderBottomWidth": null,
                "borderColor": "#c2ffcd",
                "borderLeftWidth": null,
                "borderRadius": 10,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": 2,
                "elevation": 2,
                "flexDirection": "row",
                "height": "80%",
                "justifyContent": "center",
                "marginVertical": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": undefined,
                "shadowOffset": undefined,
                "shadowOpacity": undefined,
                "shadowRadius": undefined,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 24,
                      "marginLeft": 16,
                      "marginRight": 16,
                      "paddingTop": 2,
                    },
                  ],
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              type="FontAwesome"
            >
              ÔÅ≥
            </Text>
          </View>
        </View>
      </View>
      <View
        accessible={true}
        focusable={true}
        forwardedRef={[Function]}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#114acf",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 2,
            "flexDirection": "row",
            "height": 45,
            "justifyContent": "center",
            "opacity": 1,
            "paddingBottom": 6,
            "paddingTop": 6,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            }
          }
          uppercase={false}
        >
          Save Budget
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
